lib/cipherstash/client/rpc.rb

```diff
@@ -270,17 +270,23 @@ def put_stream(collection, records)
         # isn't the most idiomatic code on earth...
         requests = [[begin_request], records].lazy.flat_map { |x| x }
 
-        res = @metrics.measure_rpc_call("putStream") do
-          stub.put_stream(requests, metadata: rpc_headers)
-        end
+        num_inserted = 0
+        @logger.info("Cipherstash::Client::RPC#put_stream") { "Start streaming upsert of documents..." }
```

Comment:
```suggestion
        @logger.debug("Cipherstash::Client::RPC#put_stream") { "Start streaming upsert of documents..." }
```

`debug` might be a more appropriate log level here.